include/deal.II/multigrid/multigrid.h

```diff
@@ -606,17 +730,24 @@ namespace internal
           const TRANSFER &transfer,
           OtherVectorType       &dst,
           const OtherVectorType &src,
-          const bool uses_dof_handler_vector,...)
+          const bool uses_dof_handler_vector,
+          const typename dealii::Multigrid<VectorType>::Signals &signals, ...)
     {
       (void) uses_dof_handler_vector;
       Assert (!uses_dof_handler_vector, ExcInternalError());
+
+      signals.transfer_to_mg(true);
       transfer.copy_to_mg(*dof_handler_vector[0],
                           multigrid.defect,
                           src);
+      signals.transfer_to_mg(false);
+
       multigrid.cycle();
+      signals.transfer_to_global(true);
```

Comment:
Add an empty line before calling the signal to make lines 746-750 a nicely delineated block.